Sources/Views/MessageInputBar.swift

```diff
@@ -339,7 +339,7 @@ open class MessageInputBar: UIView {
     private func setupConstraints() {
         
         // The constraints within the MessageInputBar
-        separatorLine.addConstraints(topAnchor, left: leftAnchor, right: rightAnchor, heightConstant: 1)
+        separatorLine.addConstraints(left: leftAnchor, bottom: topStackView.topAnchor, right: rightAnchor)
```

Comment:
Thanks for your fix, please update the commit to just remove `height` constraint. Because `topStackView` has `topStackViewPadding`, we need to tie the `separator` to the `top` of `messageInputBar`.
`separatorLine.addConstraints(topAnchor, left: leftAnchor, right: rightAnchor)`